### PR TITLE
Integrate bark.cpp

### PR DESCRIPTION
**Pull request by vinovo**

### CODE REVIEW

**Comment by zhiyuan8** [comment]
Where are the cython interfaces ?

File: .gitmodules [hunk 1]
```diff
@@ -10,3 +10,6 @@
 	path = nexa/eval/benchmark_tasks
 	url = https://github.com/NexaAI/benchmark-tasks.git
 	branch = main
+[submodule "dependency/bark.cpp"]
+	path = dependency/bark.cpp
+	url = https://github.com/NexaAI/bark.cpp.git
```
Comment (JoyboyBrian) [inline .gitmodules:15]: add `branch=main` to solve the dependency issue

Comment (vinovo) [reply .gitmodules:15]: For now it is indeed not depending on the bark.cpp main branch, but a custom branch.. I will switch to main once the bark.cpp changes are merged to main.

**Comment by vinovo** [comment]
> Where are the cython interfaces ?

@zhiyuan8 python binding interfaces? Those should be easy to add once build wheel succeed. I will add them soon.

**Comment by zhiyuan8** [comment]
@vinovo Yes, I mean, python binding for C++. Please add the python binding as well in this CL
Besides, the CI does not pass:
```
Error: fatal: Fetched in submodule path 'dependency/bark.cpp', but it did not contain 26079bb84129492464af070dada43adf8396321f. Direct fetching of that commit failed.
```

**Comment by vinovo** [comment]
> @vinovo Yes, I mean, python binding for C++. Please add the python binding as well in this CL Besides, the CI does not pass:
> 
> ```
> Error: fatal: Fetched in submodule path 'dependency/bark.cpp', but it did not contain 26079bb84129492464af070dada43adf8396321f. Direct fetching of that commit failed.
> ```

That is likely to due to the commit hash is from a non-main branch on the bark.cpp repo. I submitted a merge request in bark.cpp. After that is merged, I think the commit hash should become valid.

**Review by zhiyuan8 — APPROVED** [review]

**Pull request merged**